src/klipse/klipse_editors.cljs

```diff
@@ -65,7 +65,16 @@
               (let [result (<! evaluation-chan)
                     results (str previous-results result)]
                 (when (some? result) ;exit if the channel is closed
-                  (setter results)
+                  (if (string? result)
+                    (setter results)
+                    (let [hasError (and (= (first result) :err))]
+                      (when hasError (setter (second result)))      ;; only report the :err messages for now
```

Comment:
we need to call `setter` even when there is no error.
The reason is that when the code evaluation succeeds, we pass `[:ok res]`

Reply:
Otherwise everything looks fine

Reply:
hi @viebel the problem with setting (setter (second result)) here when there is no error is that the receiver throws 
<img width="420" alt="screen shot 2018-02-15 at 7 46 06 pm" src="https://user-images.githubusercontent.com/144226/36291629-74c39ba6-1289-11e8-9772-08f9a5a2753b.png">

I think is is because the message passed in from 
`klipse/lang/python.cljs:33`

`(put! c [:ok mod])` 

puts something on that channel that cannot be stringified.

Reply:
Let me know if that makes sense.

Reply:
you have to call `str` before putting on the channel.
```
(put! c [:ok (str mod)])
```

Reply:
Makes sense but let me show you what im running into now, it might make more sense to you..
![feb-15-2018 23-18-01](https://user-images.githubusercontent.com/144226/36295717-885b9264-12a6-11e8-951e-0ba41a20aa72.gif)

Reply:
That's with `(put! c [:ok (str mod)])`